### PR TITLE
Update tidb-lightning-faq.md from OnCALL

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Updated possible reasons for `checksum failed: checksum mismatched remote vs local`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/7197
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-faq.md [hunk 1]
```diff
@@ -223,22 +223,21 @@ Try the latest version! Maybe there is new speed improvement.
...
 
-2. If the remote checksum is 0, which means nothing is imported, it is possible that the cluster is too hot and fails to take in any data.
-
-3. If the data is mechanically generated, ensure it respects the constrains of the table:
-
-    * `AUTO_INCREMENT` columns need to be positive, and do not contain the value "0".
-    * The UNIQUE and PRIMARY KEYs must have no duplicated entries.
-
-4. If TiDB Lightning has failed before and was not properly restarted, a checksum mismatch may happen due to data being out-of-sync.
+- If `x` is greater, it means that there are more KV pairs in the import cluster.
```
Comment (glorv) [inline tidb-lightning/tidb-lightning-faq.md:230]:
```suggestion
- If `x` is greater, it means that there are more KV pairs in the target cluster.
```
should be `target` or `imported` here🤔

File: tidb-lightning/tidb-lightning-faq.md [hunk 2]
```diff
@@ -223,22 +223,21 @@ Try the latest version! Maybe there is new speed improvement.
...
+    - It is possible that this table is populated before the import and therefore affects the data checksum. It is also possible that TiDB Lightning has previously failed and shut down, but did not restart correctly.
```
Comment (glorv) [inline tidb-lightning/tidb-lightning-faq.md:231]:
```suggestion
    - It is possible that this table is not empty before the import and therefore affects the data checksum. It is also possible that TiDB Lightning has previously failed and shut down, but did not restart correctly.
```

**Review by glorv — COMMENTED** [review]
rest LGTM

**Review by glorv — COMMENTED** [review]
LGTM

File: tidb-lightning/tidb-lightning-faq.md [hunk 3]
```diff
@@ -223,22 +223,21 @@ Try the latest version! Maybe there is new speed improvement.
 
 ## `checksum failed: checksum mismatched remote vs local`
 
-**Cause**: The checksum of a table in the local data source and the remote imported database differ. This error has several deeper reasons:
+**Cause**: The checksum of a table in the local data source and the remote imported database differ. This error has several deeper reasons. You can further locate the reason by checking the log that contains `checksum mismatched`.
 
-1. The table might already have data before. These old data can affect the final checksum.
+The lines that contain `checksum mismatched` provides the information `total_kvs: x vs y`, where `x` indicates the number of key-value pairs (KV pairs) calculated by the import cluster after the import is completed, and `y` indicates the number of key-value pairs generated by the local data source.
```
Comment (lance6716) [inline tidb-lightning/tidb-lightning-faq.md:228]:
```suggestion
The lines that contain `checksum mismatched` provides the information `total_kvs: x vs y`, where `x` indicates the number of key-value pairs (KV pairs) calculated by the imported cluster after the import is completed, and `y` indicates the number of key-value pairs generated by the local data source.
```

maybe :joy: or "target cluster"

File: tidb-lightning/tidb-lightning-faq.md [hunk 4]
```diff
@@ -223,22 +223,21 @@ Try the latest version! Maybe there is new speed improvement.
...
-
-    * `AUTO_INCREMENT` columns need to be positive, and do not contain the value "0".
-    * The UNIQUE and PRIMARY KEYs must have no duplicated entries.
-
-4. If TiDB Lightning has failed before and was not properly restarted, a checksum mismatch may happen due to data being out-of-sync.
+- If `x` is greater, it means that there are more KV pairs in the target cluster.
+    - It is possible that this table is not empty before the import and therefore affects the data checksum. It is also possible that TiDB Lightning has previously failed and shut down, but did not restart correctly.
+- If `y` is greater, it means that there are more KV pairs in the local data source.
+    - If the checksum of the target database is all 0, it means that no import has occurred. It is possible that the cluster is too busy to receive any data.
+    - It is possible that the exported data contains duplicate data, such as unique and primary keys (UNIQUE and PRIMARY KEYs) with duplicate values, or that the downstream table structure is case-insensitive while the data is case-sensitive.
```
Comment (lance6716) [inline tidb-lightning/tidb-lightning-faq.md:234]:
```suggestion
    - It is possible that the exported data contains duplicate data, such as UNIQUE and PRIMARY KEYs with duplicate values, or that the downstream table structure is case-insensitive while the data is case-sensitive.
```

in Chinese I want to make sure the terminology, in English I think it's no need

**Review by lance6716 — APPROVED** [review]
LGTM but need address comment

File: tidb-lightning/tidb-lightning-faq.md [hunk 5]
```diff
@@ -223,22 +223,21 @@ Try the latest version! Maybe there is new speed improvement.
 
 ## `checksum failed: checksum mismatched remote vs local`
 
-**Cause**: The checksum of a table in the local data source and the remote imported database differ. This error has several deeper reasons:
+**Cause**: The checksum of a table in the local data source and the remote imported database differ. This error has several deeper reasons. You can further locate the reason by checking the log that contains `checksum mismatched`.
 
-1. The table might already have data before. These old data can affect the final checksum.
+The lines that contain `checksum mismatched` provides the information `total_kvs: x vs y`, where `x` indicates the number of key-value pairs (KV pairs) calculated by the target cluster after the import is completed, and `y` indicates the number of key-value pairs generated by the local data source.
```
Comment (Liuxiaozhen12) [inline tidb-lightning/tidb-lightning-faq.md:228]:
```suggestion
The lines that contain `checksum mismatched` provide the information `total_kvs: x vs y`, where `x` indicates the number of key-value pairs (KV pairs) calculated by the target cluster after the import is completed, and `y` indicates the number of key-value pairs generated by the local data source.
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 6]
```diff
@@ -223,22 +223,21 @@ Try the latest version! Maybe there is new speed improvement.
...
-
-    * `AUTO_INCREMENT` columns need to be positive, and do not contain the value "0".
-    * The UNIQUE and PRIMARY KEYs must have no duplicated entries.
-
-4. If TiDB Lightning has failed before and was not properly restarted, a checksum mismatch may happen due to data being out-of-sync.
+- If `x` is greater, it means that there are more KV pairs in the target cluster.
+    - It is possible that this table is not empty before the import and therefore affects the data checksum. It is also possible that TiDB Lightning has previously failed and shut down, but did not restart correctly.
+- If `y` is greater, it means that there are more KV pairs in the local data source.
+    - If the checksum of the target database is all 0, it means that no import has occurred. It is possible that the cluster is too busy to receive any data.
+    - It is possible that the exported data contains duplicate data, such as UNIQUE and PRIMARY KEYs with duplicate values, or that the downstream table structure is case-insensitive while the data is case-sensitive.
```
Comment (Liuxiaozhen12) [inline tidb-lightning/tidb-lightning-faq.md:234]:
```suggestion
    - It is possible that the exported data contain duplicate data, such as the UNIQUE and PRIMARY KEYs with duplicate values, or that the downstream table structure is case-insensitive while the data is case-sensitive.
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 7]
```diff
@@ -223,22 +223,21 @@ Try the latest version! Maybe there is new speed improvement.
...
+- Other possible reasons
+    - If the data source is machine-generated and not backed up by Dumpling, make sure the data conforms to the table limits, e.g., the AUTO_INCREMENT column needs to be positive and not 0.
```
Comment (Liuxiaozhen12) [inline tidb-lightning/tidb-lightning-faq.md:236]:
```suggestion
    - If the data source is machine-generated and not backed up by Dumpling, make sure the data conform to the table limits. For example, the AUTO_INCREMENT column needs to be positive and not 0.
```

**Pull request merged**